stanford/ec2_cache/README

```diff
@@ -1,3 +0,0 @@
-This temp directory created here so that we can make sure it doesn't
```

Comment:
We still cache in the same working directory (`./.cache`),
but I don't think this needs to be parameterized.